olm/olmconfig.yaml

```diff
@@ -0,0 +1,42 @@
+---
+annotations:
+  capabilityLevel: Basic Install
+  shortDescription: AWS EKS controller is a service controller for managing EKS resources
+    in Kubernetes
+displayName: AWS Controllers for Kubernetes - Amazon EKS
+description: |-
+  Manage Elastic Kubernetes Service (EKS) resources in AWS from within your Kubernetes cluster.
+
+
+  **About Amazon EKS**
+
+
+  Amazon Elastic Kubernetes Service (Amazon EKS) is a managed service that you can use to run Kubernetes on AWS without needing to install, operate, and maintain your own Kubernetes control plane or nodes. Kubernetes is an open-source system for automating the deployment, scaling, and management of containerized applications.
+
```

Comment:
Not sure if this was generated, but can we update this to say PCA instead of EKS

Reply:
+1, this file needs to be ACMPCA specific

Reply:
+1 echoing this